joomla/joomla-cms
# [4.0] Increase API Test Coverage

Continues work on adding tests for our APIs in J4

### Testing instructions
Try creating a category for a extension that is *not* com_content (contact or banners). Before patch you’ll get an error that the workflow is missing (despite banners doesn’t have workflows). After patch you can correctly create a category.

This PR also adds many api cases for banners, contacts and category api tests for content check that the api tests pass in drone
